tools/scripts-config-react-webpack/config/webpack.config.umd.js

```diff
@@ -27,6 +27,8 @@ module.exports = options => {
 	const userSassData = options.getUserConfig('sass', {});
 	const sassData = getSassData(userSassData);
 	const isEnvProd = options.mode === 'production';
+	const isEnvDevelopmentServe =
+		options.mode === 'development' && process.env.WEBPACK_SERVE === 'true';
```

Comment:
where WEBPACK_SERVE come from ?

Reply:
It's an env variable , described here : https://webpack.js.org/api/cli/#environment-options

Reply:
<img width="559" alt="image" src="https://user-images.githubusercontent.com/16574861/214797939-85b37650-0a49-4114-a148-ad1a11b96e45.png">

Reply:
this is the UMD config, so it is never used with serve

Reply:
Ho yes that's right , i can just pass false